packages/desktop-electron/window-state.ts

```diff
@@ -1,13 +1,22 @@
-const fs = require('fs');
-const path = require('path');
+import fs from 'fs';
+import path from 'path';
 
-const electron = require('electron');
+import electron, { BrowserWindow } from 'electron';
 
-// eslint-disable-next-line import/extensions
-const backend = require('loot-core/lib-dist/bundle.desktop.js');
+const backend = undefined;
+const getBackend = async () =>
```

Comment:
Has to load this in async with "import" due to 

![image](https://github.com/actualbudget/actual/assets/5285928/b3ef272c-e816-4ef4-9698-bcce46d2bc33)

So settled for loading it in like this